tagstudio/src/qt/helpers/custom_runnable.py

```diff
@@ -7,13 +7,13 @@
 
 
 class CustomRunnable(QRunnable, QObject):
-    done = Signal()
+    done = Signal(object)
```

Comment:
why object?

Reply:
> CustomRunnable needs the
ability to return the int value. Since CustomRunnable is designed to be
generic, the signal returns of a value of any `object` type.